### PR TITLE
fix publish operation timeout exception during publishing

### PR DESCRIPTION
**Pull request by cristipogacean**
* fix for #1492
* ensure the configured opc stack transport arguments (OperationTimeout, LifeTimeCount, MaxKeepAliveCount) are passed through to the opc ua stack
* removed operation timeout from the connection model - it was never propagating to the DefaultSessionManager anyhow.

### CODE REVIEW

File: components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs [hunk 1]
```diff
@@ -658,17 +691,19 @@ internal sealed class SubscriptionWrapper : ISubscription {
...
                         _logger.Error("Failed to add subscription '{name}' to session '{session}'",
                              Id, session.SessionName);
                         subscription = null;
                     }
                     else {
-                        subscription.Create();
-                        //TODO - add logs for the revised values
                         _logger.Debug("Added subscription '{name}' to session '{session}'",
                              Id, session.SessionName);
+                        subscription.Create();
```
Comment (karok2m) [inline components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs:705]: Should we have the log after we manages to create the subscription ?

File: components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs [hunk 2]
```diff
@@ -538,64 +541,66 @@ internal sealed class SubscriptionWrapper : ISubscription {
...
-                        var itemsToChange = changeList.Select(t => t.Item).ToList();
-                        var results = rawSubscription.SetMonitoringMode(change.Key.Value, itemsToChange);
-                        if (results != null) {
-                            var erroneousResultsCount = results
-                                .Count(r => (r == null) ? false : StatusCode.IsNotGood(r.StatusCode));
 
-                            // Check the number of erroneous results and log.
-                            if (erroneousResultsCount > 0) {
-                                _logger.Warning("Failed to set monitoring for {count} items in subscription "
+                        if (activate) {
```
Comment (karok2m) [inline components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs]: Please move the logic under this `if` out of `else` case on line `543`.

File: components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs (hunk 1 above)
Comment (cristipogacean) [reply components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs:705]: fixing this in another PR

File: components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs (hunk 2 above)
Comment (cristipogacean) [reply components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs]: fixed

**Review by karok2m — APPROVED** [review]

**Pull request merged**